offsets.json

```diff
@@ -0,0 +1,8 @@
+{
+    "MQDebug": {
+        "bankdefs": [1281424, 128],
+        "fontdefs": [1278576, 624],
+        "fontmaps": [1279200, 448],
+        "seqdefs": [1279648, 1776]
+    }
+}
```

Comment:
Same as `fixups.json`, I'm not sure if there's any need for a separate file really?
If their contents is expected to grow significantly with more game versions, I would at least suggest placing this and `fixups.json` in `tools/` and possibly merging them together with a name that clearly shows they're for audio

Reply:
Yeah, I can move these files into disassemble_sound.py for now until we support multiple versions.